mitsurunner.yaml

```diff
@@ -97,6 +109,28 @@ sensor:
     state_topic: $topic_outdoor
     filters:
       - filter_out: nan
+      - sliding_window_moving_average:
+          window_size: 360  #30min
+          send_every: 6     #30s
+          send_first_at: 2  #10s
```

Comment:
These are dependent on time schedules. Could there be calculation based?
Not a blocker now